<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="tabHome">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;EHM 2007 Hardcode Editor&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;The EHM Hardcode Editor is designed to edit the various rules and text stored within the EHM exe. The Editor is only compatible with the v3.0.4 eLicense version of the exe. You will encounter an error if you attempt to load any other version or a cracked version.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;It is essential that you make a backup of your ehm2007.exe file before you commence editing.&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; It does not matter what you call your edited EHM exe file; the game is happy to run an exe with any name. Hence I recommend that you have different copies of your exe for different purposes (and therefore leave your original ehm2007.exe unedited).&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;I strongly recommend you make regular backups and tests of your edited exe. The Editor is still a beta and has received very little testing. There is every chance that there may be a mistake or two in the Editor's settings, which may result in your exe crashing and/or your saved games becoming corrupt. This is why it is so important to regularly backup and test your exes before starting a proper game with them.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;If you have any questions or comments, please visit &lt;/span&gt;&lt;a href=&quot;http://www.ehmtheblueline.com/ehmupdater/&quot;&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline; color:#0000ff;&quot;&gt;http://www.ehmtheblueline.com/ehmhardcode/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QLabel" name="logoContainer">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string>Logo Container</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFolderPaths">
       <attribute name="title">
        <string>Folder Paths</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabCompNames">
       <attribute name="title">
        <string>League Names</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabCompRules">
       <attribute name="title">
        <string>League Rules</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabCompStructs">
       <attribute name="title">
        <string>League Structures</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabUserGuide">
       <attribute name="title">
        <string>User's Guide</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionVisit_EHM_Hardcode_Editor_Forum"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About EHM Hardcode Editor</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionVisit_EHM_Hardcode_Editor_Forum">
   <property name="text">
    <string>Visit EHM Hardcode Editor Forum</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
